SpatialConnect/SCJavascriptBridgeAPI.m

```diff
@@ -432,4 +439,36 @@ - (void)mqttConnected:(id<RACSubscriber>)subscriber {
       }];
 }
 
+- (void)addStore:(NSDictionary *)storeConfig
+    responseSubscriber:(id<RACSubscriber>)subscriber {
+  SCConfigService *cs = [[SpatialConnect sharedInstance] configService];
+  SCDataService *ds = [[SpatialConnect sharedInstance] dataService];
+  SCConfig *cachedConfig = cs.cachedConfig;
+  SCStoreConfig *config =
+      [[SCStoreConfig alloc] initWithDictionary:storeConfig];
+  [cachedConfig addStore:config];
+  [ds registerAndStartStoreByConfig:config];
```

Comment:
This should probably return an observable with success or fail. Write now everything about this method is synchronous but its being passed an observable. Its probably best to send some info back if the store starts successfully.

Reply:
Good point about addStore, but deleteAllDataStores and deleteDataStore probably shouldn't need an observable.  thoughts?

Reply:
Actually the addStore doesn't need to return anything bout the status, the client subscribes to the stores observable and filters on storeId to act according to the store status.

Reply:
Right, I'm just saying if the call is synchronous its confusing to have an observable. Especially if you aren't going to emit values or errors over the observable.